src/interflop-stdlib/fma/interflop_fma.c

```diff
@@ -0,0 +1,17 @@
+#include <quadmath.h>
```

Comment:
Can we please use the shim header at `test_bitmap_backend/pfp128.h` ? Libquadmath is not available on aarch64. So we should use the fma implementation in the shim and avoid linking with lquadmath for arm64.